src/Microsoft.AspNetCore.Sockets.Client.Http/LongPollingTransport.cs

```diff
@@ -53,7 +52,8 @@ public Task StartAsync(Uri url, IDuplexPipe application, TransferFormat transfer
             Log.StartTransport(_logger, transferFormat);
 
             // Start sending and polling (ask for binary if the server supports it)
-            _poller = Poll(url, _transportCts.Token);
+            var startTcs = new TaskCompletionSource<object>();
```

Comment:
Ah almost forgot this, this should look like:

```C#
new TaskCompletionSource<int>(TaskCreationOptions.RunContinuationsAsynchronously)
```

We don't want to run user code on our threads.